Makefile

```diff
@@ -18,16 +18,21 @@ vet:
 VENDOR_VERSION = v1.2.1
 vendor: vendor-$(VENDOR_VERSION)
 
-bin/bootkube: $(GOFILES) pkg/asset/internal/templates.go
+build: $(GOFILES)
+	go build -o bin/bootkube github.com/coreos/bootkube/cmd/bootkube
+
+bin/bootkube: $(GOFILES) templates
 	mkdir -p bin
 	go build -o bin/bootkube github.com/coreos/bootkube/cmd/bootkube
 
 install: all
 	cp bin/bootkube $(GOPATH_BIN)
 
-pkg/asset/internal/templates.go: $(GOFILES)
-	mkdir -p $(dir $@)
-	go generate pkg/asset/templates_gen.go
+templates:
+	go-bindata -pkg binassets -o pkg/asset/binassets/templates.go -prefix pkg/asset/templates pkg/asset/templates/...
```

Comment:
I'm not super enthusiastic about requiring a third-party tool to build the project...